### PR TITLE
Standard config for orchestration service

### PR DESCRIPTION
**Pull request by gordonfarrell**
Howdy howdy, here's a draft of a standard config/base for a post request into the orchestration service.

One question already: Did I grab the right defaults for the fhir converter template and message type?

Assumptions I was working off of writing this:
- All sensitive things such as smarty creds are already env vars available in the service itself and don't need to be sent.
- The service will be holding onto the returns from each BB and pulling from them for the next post to a BB, e.g. the FHIR bundle returned from conversion will be passed to the post used for message parsing etc.
- Most things with a default I left off assuming it would default to the default, or gave an empty string

Stuff this is not yet built to do:
- Selection of which BBs to run and in what order.
- Make omelets.

Happy to take any and all feedback if folks were imagining something very different than this

### CODE REVIEW

File: assets/sample-orchestration-request.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "xml_message": "",
+  "xml_message_type": "ecr",
```
Comment (emmastephenson) [inline assets/sample-orchestration-request.json:3]: I'm curious about this one - what are you envisioning `xml_message`/`xml_message_type` will be used for?

File: assets/sample-orchestration-request.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "message_type": "ecr"
+    },
+    "standardization_and_geocoding": {
+      "geocode_method": "smarty",
+      "overwrite": "true",
+      "dob_format": ""
+    },
+    "message_parser": {
+      "message_format": "fhir",
+      "parsing_schema": "",
```
Comment (emmastephenson) [inline assets/sample-orchestration-request.json:19]: I don't completely remember how the message parser works 😅 If `parsing_schema` is left empty, will it default to the one we use in LAC?

**Review by emmastephenson — COMMENTED** [review]
This looks good Gordon, thank you! Just a couple of questions.

File: assets/sample-orchestration-request.json (hunk 1 above)
Comment (nickclyde) [reply assets/sample-orchestration-request.json:3]: I'm guessing `xml_message` is the field for the raw message to be sent into the FHIR converter? Though we also support HL7 messages which are pipe delimited files, so maybe it should just be `message` and `message_type`?

Comment (emmastephenson) [reply assets/sample-orchestration-request.json]: Ahhhh gotcha, that makes sense! I think in that case my vote would be to make those part of the request params for our `parse` endpoint, vs part of the configuration. Thanks for explaining!

File: assets/sample-orchestration-request.json (hunk 2 above)
Comment (emmastephenson) [reply assets/sample-orchestration-request.json:18]: Gotcha! That's always something we can edit later too, as we work to add functionality to the service.

**Review by emmastephenson — APPROVED** [review]
LGTM! Thanks Gordon!

**Pull request merged**